src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/Process/Process.ts

```diff
@@ -27,13 +27,14 @@ export interface IStateData {
 
 export interface IProcessProperties {
     detailSlot : string;
+    document? : boolean;
```

Comment:
I would prefer to continue with the slot concept, so add `editSlot` and `createSlot`.